deps.windows/50-pcre.ps1

```diff
@@ -25,7 +25,6 @@ function Configure {
 
     $Options = @(
         $CmakeOptions
-        "-DBUILD_SHARED_LIBS:BOOL=OFF"
```

Comment:
What's the reason here?

Reply:
Looks like that one sneaked in through a cherry-pick, removed it with a force-push.